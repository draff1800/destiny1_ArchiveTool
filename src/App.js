import React from 'react';
import './App.css';
import 'antd/dist/antd.css';
import { Button, Typography } from 'antd'; 

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <Typography.Title level={1} style={styles.title}>
          We need your permission...
        </Typography.Title>
        <Typography.Text style={styles.text}>
          Allow DAT to view your Destiny characters.
        </Typography.Text>
        <Button type="primary" size="large" style={styles.button}>Authorise with Bungie.net</Button>
      </header>
    </div>
  );
}

const styles = {
  title: {color: "white", fontWeight: "normal", marginBottom: 5},
  text: {fontSize: 18, color: "white", marginBottom: 20},
  button: {backgroundColor: "orange", color: "#353535", fontWeight: "bold"}
}

export default App;
