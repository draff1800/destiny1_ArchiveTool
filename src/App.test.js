// Link.react.test.js
import React from 'react';
import App from './App';
import renderer from 'react-test-renderer';

test('Check that the page renders', () => {
  const component = renderer.create(
    <App></App>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
