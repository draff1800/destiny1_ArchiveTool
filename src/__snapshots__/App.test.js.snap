// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check that the page renders 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <h1
      className="ant-typography"
      style={
        Object {
          "WebkitLineClamp": null,
          "color": "white",
          "fontWeight": "normal",
          "marginBottom": 5,
        }
      }
    >
      We need your permission...
    </h1>
    <span
      className="ant-typography"
      style={
        Object {
          "WebkitLineClamp": null,
          "color": "white",
          "fontSize": 18,
          "marginBottom": 20,
        }
      }
    >
      Allow DAT to view your Destiny characters.
    </span>
    <button
      className="ant-btn ant-btn-primary ant-btn-lg"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "orange",
          "color": "#353535",
          "fontWeight": "bold",
        }
      }
      type="button"
    >
      <span>
        Authorise with Bungie.net
      </span>
    </button>
  </header>
</div>
`;
